src/fmllauncher/java/net/minecraftforge/common/asm/RuntimeEnumExtender.java

```diff
@@ -150,6 +150,16 @@ public int processClassWithFlags(final Phase phase, final ClassNode classNode, f
 
             mtd.access |= Opcodes.ACC_SYNCHRONIZED;
             mtd.instructions.clear();
+            mtd.localVariables.clear();
+            if (mtd.tryCatchBlocks != null) {
+                mtd.tryCatchBlocks.clear();
+            }
+            if (mtd.visibleLocalVariableAnnotations != null) {
+                mtd.visibleLocalVariableAnnotations.clear();
+            }
+            if (mtd.invisibleLocalVariableAnnotations != null) {
+                mtd.invisibleLocalVariableAnnotations.clear();
+            }
```

Comment:
Formatting: braces on newlines.
```suggestion
            if (mtd.tryCatchBlocks != null)
            {
                mtd.tryCatchBlocks.clear();
            }
            if (mtd.visibleLocalVariableAnnotations != null)
            {
                mtd.visibleLocalVariableAnnotations.clear();
            }
            if (mtd.invisibleLocalVariableAnnotations != null)
            {
                mtd.invisibleLocalVariableAnnotations.clear();
            }
```